executor/internal/calibrateresource/calibrate_resource.go

```diff
@@ -391,6 +459,22 @@ func (e *Executor) staticCalibrate(ctx context.Context, req *chunk.Chunk, exec s
 	return nil
 }
 
+func staticCalibrateTpch10(ctx context.Context, req *chunk.Chunk, exec sqlexec.RestrictedSQLExecutor, ruCfg *resourceControlClient.RUConfig) error {
+	// TiFlash only consider cpu and read bytes. Others are ignored.
+	// cpu usage: 105494.666484 / 20 / 20 = 263.74
+	// read bytes: 401799161689.0 / 20 / 20 = 1004497904.22
+	const cpuTimePerCPUPerSec float64 = 263.74
+	const readBytesPerCPUPerSec float64 = 1004497904.22
+	ruPerCPU := float64(ruCfg.CPUMsCost)*cpuTimePerCPUPerSec + float64(ruCfg.ReadBytesCost)*readBytesPerCPUPerSec
```

Comment:
An additional question: Is the cost factor used by Tiflash to calculate RU obtained from PD?

Reply:
No, the number is from tiflash log(105494.666484 and 401799161689.0)